src/libraries/restore/runtime/runtime.depproj

```diff
@@ -71,6 +74,11 @@
           SkipUnchangedFiles="true"
           UseHardlinksIfPossible="$(UseHardlink)" />
 
+    <Copy SourceFiles="@(HostPolicyFile)"
```

Comment:
Condition to `RuntimeFlavor == Mono`? I don't know how Copy behaves with empty items.

Reply:
Good point.  I assumed it would skip.